packages/styles/scss/patterns/blocks/leadspace-block/index.scss

```diff
@@ -0,0 +1,4 @@
+
+
+@import '../../../globals/imports';
+@import 'leadspace-block';
```

Comment:
Is this file being referenced at all? Seems like importing `leadspace-block` directly instead, since that has all the imports needed.

Reply:
yeah, removed it.